build_dir/Cargo.toml

```diff
@@ -26,6 +26,7 @@ arrow-buffer = {git = 'https://github.com/ArroyoSystems/arrow-rs', branch = '46.
 arrow-array = {git = 'https://github.com/ArroyoSystems/arrow-rs', branch = '46.0.0/parquet_bytes'}
 arrow-schema = {git = 'https://github.com/ArroyoSystems/arrow-rs', branch = '46.0.0/parquet_bytes'}
 object_store = {git = 'https://github.com/ArroyoSystems/arrow-rs', branch = 'object_store/put_part_api'}
+deltalake = {git = "https://github.com/ArroyoSystems/delta-rs", branch = "dynamo_pin_0_16"}
```

Comment:
do we also need these changes in the docker images?

Reply:
Yeah, I will, otherwise our builds will fail. One sec.